content/applications/inventory_and_mrp/purchase/products/pricelist.rst

```diff
@@ -0,0 +1,260 @@
+
+To manually add the vendor price on the product form, go to the :menuselection:`Purchase app -->
+Products --> Products`, and click the desired product.
+
+.. note::
+   Product forms are accessible from multiple apps, such as **Sales**, **Inventory**, and
+   **Manufacturing**.
+
+In the :guilabel:`Purchase` tab of the product form, input the vendor and their price, to have this
+information auto-populate on a request for quotation each time the product is listed.
+
+.. seealso::
+   :ref:`Vendor pricelist on product form <purchase/manage_deals/vendor-pricelist>`
+
+.. image:: pricelist/product-form-pricelist.png
+   :align: center
+   :alt: Vendor pricelist on product form.
+
+.. _purchase/products/import-pricelist:
+
+Import vendor pricelist
+=======================
+
+To import vendor pricelists, ensure the XLSX or CSV file is accurately completed. The best way to
+obtain a correctly formatted template, including product names, references, and vendor details, is
+to first :ref:`export a pricelist <purchase/products/export-price>` from the database.
+
+Modify the exported file, as needed, then import it back into the Odoo database.
+
+.. _purchase/products/export-price:
+
+Export pricelist
+----------------
+
+To export a pricelist, go to :menuselection:`Purchase app --> Configuration --> Vendor Pricelists`.
+
+On the page, tick the checkbox(es) for the desired vendor pricelists.
+
+Then, click the :icon:`fa-cog` :guilabel:`Actions` button that appears, and choose :icon:`fa-upload`
+:guilabel:`Export` from the drop-down menu.
+
+.. image:: pricelist/export.png
+   :align: center
+   :alt: Show selected exported fields, with the Export button visible.
+
+In the resulting pop-up window, fields listed under the :guilabel:`Fields to export` section are
+included in the exported file. To add more fields, find the desired field in the
+:guilabel:`Available fields` section, and click the :icon:`fa-plus` :guilabel:`(plus)` icon to the
+right of the field.
+
+.. note::
+   To update to existing records, tick the :guilabel:`I want to update data (import-compatible export)` checkbox, and refer to
+   the section on the :ref:`External ID <purchase/products/external-id>` field.
+
+   For details on commonly-used fields for importing vendor pricelists, see the :ref:`Common fields
+   <purchase/products/common-fields>` section.
+
+Select the desired :guilabel:`Export Format`: :guilabel:`XLSX` or :guilabel:`CSV`.
+
+To save the selected fields as a template, click the :guilabel:`Template` field, and select
+:guilabel:`New template` from the drop-down menu. Type the name of the new template, and click the
+:icon:`fa-floppy-o` :guilabel:`(save)` icon. After that, the template is a selectable option
+when clicking the :guilabel:`Template` field.
+
+Finally, click :guilabel:`Export`.
+
+.. note::
+   With :ref:`developer mode <developer-mode>` turned on, the column names of the exported file
+   display the *Technical name of the field*, not the *Field name*.
```

Comment:
The field name is still shown, with the tech name in () :)

Also I dont see a need to capitalize these, feel free to ignore me if KSC suggested this
```suggestion
   display the *field name* with the *technical name* in parenthesis.
```